### PR TITLE
Missing Binary data in Configmap feature in documentation

### PR DESCRIPTION
**Pull request by mburke5678**
https://bugzilla.redhat.com/show_bug.cgi?id=1627755

### CODE REVIEW

**Comment by mburke5678** [comment]
@smarterclayton [PTAL at my questions above](https://github.com/openshift/openshift-docs/pull/12099#issuecomment-422945247).

**Comment by mburke5678** [comment]
@mfojtik @mcurry-rh PTAL

**Comment by mburke5678** [comment]
@mfojtik @mcurry-rh BUMP

**Comment by mburke5678** [comment]
@sferich888 @mfojtik @mcurry-rh 

Can anyone help with [Eric's question 2 and 3 above](https://github.com/openshift/openshift-docs/pull/12099#issuecomment-431067517)?  
I can give it a try if anyone has a binary file to use. I downloaded a couple, but got errors when using those file. 
What is the end result of including the binary files?

**Comment by mburke5678** [comment]
@sferich888 8 @mfojtik  @mcurry-rh  BUMP

**Comment by zhouying7780** [comment]
@wjiangjay  PTAL, thanks.

**Comment by wjiangjay** [comment]
Checked and LGTM

**Comment by mburke5678** [comment]
Per eric rich email: While I don't like those questions being answered, what you have looks good.

**Comment by mburke5678** [comment]
@openshift/team-documentation PTAL

File: dev_guide/configmaps.adoc [hunk 1]
```diff
@@ -47,8 +47,17 @@ data: <1>
     property.1=value-1
     property.2=value-2
     property.3=value-3
+binaryData:
+  bar: L3Jvb3QvMTAw <2>
 ----
 <1> Contains the configuration data.
+<2> Points to a file containing non-UTF8 data. For example, a binary Java keystore file.
```
Comment (kalexand-rh) [inline dev_guide/configmaps.adoc]: I'd s/data. For/ data, for and s/containing/that contains

File: dev_guide/configmaps.adoc [hunk 2]
```diff
@@ -169,6 +178,12 @@ You can also pass the `--from-file` option with a specific file, and pass it
 multiple times to the CLI. The following yields equivalent results to the
 xref:configmaps-creating-from-directories[Creating from Directories] example:
 
+[NOTE]
+====
+If you create a configmap from a file, you can include files containing non-UTF8 data will be placed in this new field in order to preserve the non-UTF8 data.
```
Comment (kalexand-rh) [inline dev_guide/configmaps.adoc]: Please double-check this sentence. Maybe s/containing non-UTF8 data will be placed/that contain non-UTF8

**Review by kalexand-rh — COMMENTED** [review]
A pick, as always, and a question, then LGTM.

File: dev_guide/configmaps.adoc (hunk 2 above)
Comment (mburke5678) [reply dev_guide/configmaps.adoc]: @kalexand-rh That sentence came from the [Kubernetes change log](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.10.md#apps).

I changed it to _on-UTF8 data will be placed in this new field without corrupting the non-UTF8 data._, taken from https://github.com/kubernetes/kubernetes/pull/57899#issue-161357094

**Pull request merged**